src/talos/ValidateMOI.py

```diff
@@ -201,6 +201,10 @@ def clean_and_filter(
 
     cohort_panels = set(config_retrieve(['GeneratePanelData', 'forced_panels'], []))
 
+    # add the custom gene panel ID as a forced panel
+    # any gene added manually, either to override MOI, or to add the gene completely, will be applied to every family
+    cohort_panels.add(0)
```

Comment:
by allocating all custom genes to panel 0, we have a way to ensure we never remove these genes from a family's results because the 'custom' panel wasn't allocated to them. Panel 0 is applied to all manual genes, and Panel 0 is applied to every family. We can surface this later in the reports as a manually applied gene